### PR TITLE
Use @lavamoat/allow-scripts

### PR DESCRIPTION
**Pull request by Mrtenz**
`@lavamoat/allow-scripts` was installed but not enabled. This adds the Yarn plugin, and adds a section to the root `package.json` to actually enable it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,10 +44,17 @@
...
     "typescript": "^4.7.4"
   },
   "packageManager": "yarn@3.2.1",
   "engines": {
     "node": ">=18.6.0"
+  },
+  "lavamoat": {
+    "allowScripts": {
+      "@lavamoat/preinstall-always-fail": false,
+      "sharp": true
```
Comment (Mrtenz) [inline package.json:57]: Sharp is enabled since it's used by Gatsby.

File: package.json [hunk 2]
```diff
@@ -27,6 +27,8 @@
     "test": "yarn workspace snap run test"
   },
   "devDependencies": {
+    "@lavamoat/allow-scripts": "^3.0.0",
```
Comment (FrederikBolding) [inline package.json:30]: Let's remove allow-scripts from the snap package then as it should be applied at the root

File: README.md [hunk 3]
```diff
@@ -1,78 +1,54 @@
...
 ## Snaps is pre-release software
 
-To interact with (your) Snaps, you will need to install [MetaMask Flask](https://metamask.io/flask/), a canary distribution for developers that provides access to upcoming features.
+To interact with (your) Snaps, you will need to install [MetaMask Flask](https://metamask.io/flask/),
+a canary distribution for developers that provides access to upcoming features.
 
 ## Getting Started
 
-Clone the template-snap repository [using this template](https://github.com/MetaMask/template-snap-monorepo/generate) and setup the development environment:
+Clone the template-snap repository [using this template](https://github.
```
Comment (FrederikBolding) [inline README.md:17]: This is not valid

File: README.md [hunk 4]
```diff
@@ -1,78 +1,53 @@
...
+If you don't wish to use any of the existing GitHub actions in this repository,
+simply delete the `.github/workflows` directory.
 
 ## Contributing
 
 ### Testing and Linting
 
 Run `yarn test` to run the tests once.
 
-Run `yarn lint` to run the linter, or run `yarn lint:fix` to run the linter and fix any automatically fixable issues.
```
Comment (FrederikBolding) [inline README.md:35]: Is this intentionally removed?

**Review by FrederikBolding — DISMISSED** [review]

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**